main.go

```diff
@@ -45,7 +45,7 @@ import (
 var (
 	scheme               = runtime.NewScheme()
 	setupLog             = ctrl.Log.WithName("setup")
-	metricsAddr          = flag.String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	metricsAddr          = flag.String("metrics-bind-address", ":8889", "The address the metric endpoint binds to.")
```

Comment:
What does this change do?

Reply:
its for #502 since runtime metrics are not a part of otel, we can't remove it but we can have them displayed with the rest of the collected metrics at the same endpoint